tests/ztimer_xsec/Makefile

```diff
@@ -1,6 +1,7 @@
 include ../Makefile.tests_common
 
 USEMODULE += ztimer
+USEMODULE += ztimer_nsec
```

Comment:
Somehow `ztimer_nsec` is not pulled in with this, but `periph_ptp_timer` is. Or do I have to add somethiing else?
Can I alternatively add a comment in `Makefile` like in the other test?

```
# uncomment this to test using ztimer nsec on ptp
#USEMODULE += ztimer_nsec ztimer_periph_ptp
```

Reply:
It could be that the dependency resolution doesn't expect a Makefile.dep within the applications folder. That should be relatively straight forward to fix, though. Just add an `-include $(THE_VARIABLE_NAME_OFF_THE_APP_DIR_THAT_I_CANT_REMEMBER)/Makefile.dep` to `makefiles/dependency_resolution.inc.mk`.

If that works, please split out as separate PR. Build system changes and especially dependency resolution changes have a tendency to break things in impossible to foresee ways...

Reply:
Currently, for applications there is a `Makefile.board.dep` for `BOARD` related dependencies, so that would the place to put this.